dao/src/main/java/greencity/repository/HabitAssignRepo.java

```diff
@@ -85,6 +85,20 @@ Optional<HabitAssign> findByHabitIdAndUserIdAndCreateDate(@Param("habitId") Long
     Optional<HabitAssign> findByHabitIdAndUserIdAndStatusIsInprogress(@Param("habitId") Long habitId,
         @Param("userId") Long userId);
 
+    /**
+     * Method to find {@link HabitAssign}'s by {@link User} and {@link HabitAssign}
+     * id and INPROGRESS status.
+     *
+     * @param habitAssignId {@link Long} id.
+     * @param userId        {@link Long} id.
+     * @return {@link HabitAssign} instance.
+     * @author Anton Bondar
+     */
+    @Query(value = "SELECT DISTINCT ha FROM HabitAssign ha"
```

Comment:
Why do we need `DISTINCT` here?
can we have several HabitAssign with the same `habitAssignId` and `userId`?